### ISSUE TITLE
svcrec fatal error

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#158]
Sometimes svcrec is dying with the following error: 

```
error: open_file - cannot open /opt/kronometrix/log//opt/kronometrix/bin/svcrec.log No such file or directory at /opt/kronometrix/perl/lib/site_perl/5.24.1/Kronometrix.pm line 58.
getaddrinfo(www.apple.com,,AF_INET) failed - hostname nor servname provided, or not known at /opt/kronometrix/perl/lib/site_perl/5.24.2/Kronometrix/Svcrec.pm line 112.
```

this might be related with DNS or not but after this the svcrec is ending its execution. Can we catch and retry within the main loop this error such way we dont die and wait for another loop ? In this case we should simple report NA for all values ...

Not sure how you could reproduce this. Im seeing this periodically in one of our test machines.

**Comment by sparvu** [comment]
make sure you git clone to fetch latest KDR image

**Comment by sparvu** [comment]
fixed in latest 1.2.26
